targets/targets.json

```diff
@@ -3364,7 +3365,7 @@
             "MPU"
         ],
         "release_versions": ["2", "5"],
-        "bootloader_supported": true
+        "bootloader_supported": false
```

Comment:
not related to the PR ?

Reply:
Good catch, definitely not. That's an early bring-up change to avoid the `MBED_ROM_START` problems. I'll force push a fix.